src/index.js

```diff
@@ -377,28 +411,62 @@ function drawPrefectureTable(prefectures, totals) {
     }else{
       prefStr = pref.name_ja
     }
+
+    let increment = pref.dailyConfirmedCount[pref.dailyConfirmedCount.length - 1]
+    let incrementString = ''
+    if (increment > 0) {
+      incrementString = `<span class='increment'>(+${increment})</span>`
+    }
     
-    // TODO Make this pretty
-    
-    if(pref.name == 'Unspecified'){
+    if (pref.name == 'Unspecified'){
       // Save the "Unspecified" row for the end of the table
-      unspecifiedRow = "<tr><td><em>" + prefStr + "</em></td><td>" + pref.confirmed + "</td><td>" + (pref.recovered?pref.recovered:'') + "</td><td>" + pref.deaths + "</td></tr>"
-    }else if (pref.name == 'Total'){
+      unspecifiedRow = `<td class="prefecture">${prefStr}</td>` +
```

Comment:
I think the new template literals work safely across multiple lines ([source](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)). Is there another reason you wanted to add `+` on each row?

Reply:
Ohh, I didn't know that. That makes it much nicer ...